fix(Note): Expose `--light-text` CSS var, fixups (#260)
getsentry/develop: src/docs/config.mdx

If logging in a particular module is not showing up when you expect it to, you should check the log level for that module in `src/sentry/conf/server.py` in the `LOGGING` variable.

> <Note><markdown>

Reviewer: I haven't found it necessary to use the <markdown></markdown> elements in user docs, FYI, but I don't think they'll cause issues.

Author: Eh, will leave them for now. May do another sweep to fix them.

> If you are a Sentry employee who is looking for logs in Kibana and they aren't showing up when you expect them to, one potential reason is that a field in the log message not matching Elasticsearch's schema. Check <a href="https://kibana.getsentry.net/app/kibana#/management/kibana/indices/0ba261d0-97ca-11e9-90db-dfacda4da93b?_g=()&_a=(tab:indexedFields)">the index patterns</a> to see whether this is the reason.

Reviewer: ```suggestion
If you are a Sentry employee looking for logs in Kibana and they aren't showing up when as expected, one potential reason is that a field in the log message does not match Elasticsearch's schema. Check <a href="https://kibana.getsentry.net/app/kibana#/management/kibana/indices/0ba261d0-97ca-11e9-90db-dfacda4da93b?_g=()&_a=(tab:indexedFields)">the index patterns</a> to see whether this is the reason.
```

Your app's private key.

<Alert level="warning">

> Make sure to replace the newlines with `\n` to preserve them.

Reviewer: ```suggestion
Replace the newlines with `\n` to preserve them.
```
